vmdk_plugin/utils/plugin_server/plugin_server_linux.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package plugin_server
```

Comment:
Sorry I don't quite understand what you mean by separate into separate packages for vmdk, windows, shared? Linux/windows plugin_server are just platform specific compilation choice for plugin servers. It's getting compiled according to the platform where you build the code. It will be automatically instantiated. I guess you may understand this part of code wrong... Please ping me offline so we can discuss more about it. @venilnoronha